docker-compose.client.yml

```diff
@@ -27,6 +27,7 @@ services:
     volumes:
       - core-storage:/data
       - reports-storage:/reports
+      - /dev:/dev
```

Comment:
What's the point of this? Bind mounting `/dev` can cause permissions issues on the host.